pkg/pb/satellite.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package satellite;
+
+import "gogo.proto";
+
+service Satellite {
+    rpc Health(FileHealthRequest) returns (FileHealthResponse) {}
+}
+  
+message FileHealthRequest {
+    string path = 1;
```

Comment:
Also separate the bucket and path, preemptively, since we'll do that split everywhere.

Reply:
done